Gui/opensim/installer_files/Linux/INSTALL

```diff
@@ -83,26 +83,31 @@ done
 ## End from https://stackoverflow.com/a/29754866
 
 if [[ -e /etc/os-release ]]; then
-    . /etc/os-release
-    OS=$NAME
-    VER=$VERSION_ID
+    . /etc/os-release 
+    OS=$ID
+    OS_NAME=$NAME
+    if ! [ -z "${VERSION_ID+x}" ]; then
+        VER=$VERSION_ID
+    fi
+    if ! [ -z "${PRETTY_NAME+x}" ]; then
+        OS_FULL=$PRETTY_NAME
+    fi
 else
     err_unsupported_env
-fi
-
-if [[ $OS == "Ubuntu" ]] && [[ $VER == "18.04" ]]; then
+fi 
+if ([[ $OS == "ubuntu" ]] && [[ $VER == "18.04" ]]) || ([[ $OS == "debian" ]] && [[ $OS_FULL =~ "bullseye" ]]); then
```

Comment:
Is there a reason (ie in terms of an expectation of a non-exact match) for using the `=~` comparison for matching "bullseye"? I'm not familiar with Debian